src/PointwiseFunctions/GeneralRelativity/Christoffel.cpp

```diff
@@ -10,7 +10,9 @@
 template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::abb<DataType, SpatialDim, Frame, Index> compute_christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) {
-  tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel{};
+  auto christoffel =
+      make_with_value<tnsr::abb<DataType, SpatialDim, Frame, Index>>(
+          get<0, 0, 0>(d_metric), 0.);
```

Comment:
you should just be able to pass d_metric instead of get<...>(d_metric);  you can pass any object that has the correctly sized underlying DataType